Mathlib/SetTheory/Ordinal/FixedPointApproximants.lean

```diff
@@ -159,7 +159,7 @@ lemma lfpApprox_mem_fixedPoints_of_eq {a b c : Ordinal}
   · exact h_ac
   · exact lfpApprox_mem_fixedPoint
 
-/-- A fixed point of f is reached after the successor of the domains cardinality -/
+/-- A fixed point of f is reached at the ordinal-successor index of the domains cardinality-/
```

Comment:
```suggestion
/-- A fixed point of f is reached at the successor of the domains cardinality-/
```
This isn't an ordinal successor, but rather a cardinal one!

Reply:
Oooops. I may have thought about a successor ordinal used in the papers, which we could avoid using because Lean is not based on ZFC.

Reply:
Also, I reordered the sentence so it is not passive any more.